docs/en/solutions/NotReady_Node_with_High_Load_and_Many_D_State_Processes.md

```
@@ -0,0 +1,50 @@
+For workloads that can tolerate an application-visible error instead of an indefinite wait, remount the affected NFS volume with the `soft` option so the client returns to userspace after `retrans` retransmissions; accept that this exposes the workload to silent data corruption on partially completed writes, which is the reason `hard` is the upstream default. The same `hard`/`soft`/`retrans` semantics apply uniformly to NFS-backed PVs on this platform: the catalog ships an NFS CSI ModulePlugin (`chart-csi-driver-nfs`, default channel `v4.4.0-beta.7`) that mounts via the standard Linux NFS client, so kernel-side mount options carry through unchanged on Kubernetes `v1.34.5` clusters [ev:c6].
+
+## Diagnostic Steps
+
+Confirm the node-level symptom from the cluster side first. `kubectl get nodes` shows `STATUS=NotReady` for the affected node, and the Ready condition on `.status.conditions[?(@.type=='Ready')]` carries the upstream NodeCondition fields (`type`, `status`, `reason`, `message`, `lastHeartbeatTime`, `lastTransitionTime`); a healthy peer prints `reason=KubeletReady` / `message=kubelet is posting ready status`, so a comparison against any Ready peer is the quickest sanity check [ev:c4]:
+
+```bash
+kubectl get nodes
+kubectl get node <node-name> -o jsonpath="{.status.conditions[?(@.type=='Ready')]}{'\n'}"
+```
+
+Open a host-level shell on the affected node through the cluster's standard node-access method — typically a `kubectl debug node/<name>` debug pod, direct SSH from the installer host, or whatever node-admin path the platform documents; the host-side diagnostic commands themselves (`uptime`, `ps -elfL`, `dmesg`) are unchanged across entry paths. From that host shell, inspect the load average and the count / WCHAN of D-state threads. A load average far above the runnable-process count combined with `ps -elfL` rows whose state column is `D` and whose `WCHAN` column reads `rpc_wa` confirms threads blocked inside the sunrpc client [ev:c1][ev:c2]:
```

markdownlint-cli2 (0.22.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "ev"

(MD052, reference-links-images)